Analytics/resources/moving_sensors/get_dummy_data.py

```diff
@@ -0,0 +1,134 @@
+                 a JSON error message is returned with the appropriate HTTPStatus code
+        """
+        args = self.reqparser.parse_args()
+        # Get size of tables before import
+        self.loc_stats["before"] = db.session.query(func.count(LocationData.id)).scalar()
+        self.tracker_stats["before"] = db.session.query(func.count(Tracker.id)).scalar()
+
+        # Fetch Data from CSV
+        try:
+            df = pd.read_csv(args['file_name'])
+        except IOError as ioe:
+            logger.error("Unable to parse CSV data to dataframe", ioe.with_traceback(ioe.__traceback__))
+            return dict(error="Unable to parse CSV data to dataframe",
+                        trackback=ioe.with_traceback(ioe.__traceback__)), HTTPStatus.BAD_REQUEST
+
+        # Trackers must be unique, Fetch trackers and make dB entries for each unique tracker
+        unique_tracker_ids = df["tracker"].unique()
+        for tracker_id in unique_tracker_ids:
+            self.t_ids[self.create_trackers(str(tracker_id))] = 0
+
+        # Define Location data Pandas DataFrame Column names
+        loc_df = df[
+            ['tracker', 'datetime', 'latitude', 'longitude', 'speed', 'heading', 'elevation', 'charger', 'battery',
+             'signalquality', 'satcnt']]
+
+        # Drop all entries that are incomplete have NaN or None/ Null values
+        loc_df = loc_df.dropna()
+
+        # Store Location Data in the dB
+        for index, row in loc_df.iterrows():
+            self.add_location_data(row['tracker'], row['datetime'], row['latitude'], row['longitude'], row['speed'],
+                                   row['heading'], row['elevation'], row['charger'], row['battery'],
+                                   row['signalquality'], row['satcnt'])
+
+        self.loc_stats["after"] = db.session.query(func.count(LocationData.id)).scalar()
+        self.tracker_stats["after"] = db.session.query(func.count(Tracker.id)).scalar()
+
+        return self.status_report(), 200
+
+    def create_trackers(self, tid: str) -> str:
+        """
+        Create new Tracker
+        :param tid: Tracker Id
+        :return: The Tracker Id
+        """
+        tracker = Tracker(tid)
+        tracker.save()
+        tracker.commit()
+        return tid
+
+    def add_location_data(self, tracker_id: str, timestamp: str, latitude: float, longitude: float, speed: float,
+                          heading: float, elevation: float, charger: bool, battery: float, signalquality: int,
+                          satcnt: int) -> None:
+        """
+        Create LocationData
+        :param tracker_id: Trackers Id
+        :param timestamp: Timestamp of when measurement was made
+        :param latitude: GPS Latitude coordinate in Decimal Degree (+-DD.ddddddd)
+        :param longitude: GPS Latitude coordinate in Decimal Degree (+-DD.ddddddd)
+        :param speed: Velocity of sensor in m per second
```

Comment:
Either type `meters per second` or `m/s`, preferably the first option. Keep things constant

Reply:
changed to meters per second